test/integration/editor/index.spec.ts

```diff
@@ -115,3 +115,20 @@ test('code editor auto-complete', async ({ page }) => {
   await page.keyboard.type('textF');
   await expect(page.getByRole('option', { name: 'textField' })).toBeVisible();
 });
+
+test('cancel the status of active when click the white space', async ({ page }) => {
```

Comment:
I'd reword the test name to something like `must deselect selected element when clicking outside of it`